README.md

```diff
@@ -0,0 +1,20 @@
+<img src="./art/repo_header.png" alt="Image Cropper for Xamarin.Android" width="728" />
+
+# Image Cropper for Xamarin.Android
+A Xamarin.Android binding library for [Image Cropper](https://github.com/ArthurHub/Android-Image-Cropper) library.
+
+```
+Install-Package XamarinAndroidImageCropper
+```
+
+## Usage
+
+Plz follow its origin guide [here](https://github.com/ArthurHub/Android-Image-Cropper/blob/master/README.md).
```

Comment:
Use proper english
"Please follow the Android library usage guide here"

Reply:
:). Thanks.